web/packages/teleterm/src/ui/DocumentCluster/UnifiedResources.tsx

```diff
@@ -340,6 +387,26 @@ const Resources = memo(
         params={props.queryParams}
         setParams={props.onParamsChange}
         unifiedResourcePreferencesAttempt={props.userPreferencesAttempt}
+        bulkActions={
+          props.integratedAccessRequests.supported === 'yes'
+            ? [
+                {
+                  key: 'requestAccess',
+                  Icon: icons.AddCircle,
+                  text: 'Request Access',
```

Comment:
In the Web UI, after I add a resource to the request, this button says "Add/Remove to Request". In Connect, it always says "Request Access". I feel like the copy should change in Connect just like it changes in the Web UI.

Reply:
Unless I'm missing something, it seemed pretty hard to get in the "count" of selectedResources at this point. I thought about moving the bulkActions up into a prop passed into `Resource`, and while it does have its merits, I decided to just make this prop be `string` or function that receives the selected resources (similar to `action` in this type).

This way I can keep all the callsites the same except for this one and I don't have to rejigger everything we've added to make it work _here_.

I will update if there is great opposition to this but its a pattern ive seen frequently enough around the place that I think its ok.